### PR TITLE
Remove expired academic discount coupon alert box

### PR DESCRIPTION
**Pull request by cham11ng**
image

### CODE REVIEW

**Comment by mesaugat** [comment]
This is critical information. Should we really hide it?

**Comment by cham11ng** [comment]
Every private repository are archived after coupon is expired as well as they get notified though emails. This information is critical who sees it first time. But for me, I've already downgraded it and it shows downgrade request is still pending so I am getting this. Also they can check in `Settings > Billing` so hiding this won't affect so far.

image

File: main.css [hunk 1]
```diff
@@ -187,3 +187,10 @@
 .dashboard .news .follow .Box.p-3.mt-2 {
   display: none;
 }
+
+/**
+ * Remove expired academic discount coupon alert box.
+ */
+.flash {
```
Comment (mesaugat) [inline main.css:194]: Does this have a parent container like `.dashboard`? If yes, we need to do something like `.dashboard .flash` in order to only hide this from the dashboard. `.flash` might affect the whole of GitHub.

**Comment by cham11ng** [comment]
It's gone now so closing this PR. I couldn't find the parent class.

**Pull request closed without merging**